lib/src/rules/unawaited_futures.dart

```diff
@@ -93,6 +105,16 @@ class Visitor extends SimpleAstVisitor {
   bool _isMapClass(Element e) =>
       e is ClassElement && e.name == 'Map' && e.library?.name == 'dart.core';
 
+  bool _isTestExpectFunction(Element e) {
+    if (e is! FunctionElement || e.name != 'expect') return false;
+
+    final libUri = e.library?.source?.uri?.toString();
+    if (libUri == null) return false;
+
+    return libUri.startsWith('package:test/') ||
+        libUri.startsWith('package:unittest/');
```

Comment:
(not that it matters anymore, but fyi most our tests are still using unittest :-))